static-assets/components/cstudio-dashboard-widgets/my-recent-activity.js

```diff
@@ -24,327 +24,324 @@ if (typeof CStudioAuthoringWidgets == 'undefined' || !CStudioAuthoringWidgets) {
 /**
  * my recent activity
  */
-CStudioAuthoringWidgets.MyRecentActivityDashboard =
-  CStudioAuthoringWidgets.MyRecentActivityDashboard ||
-  function (widgetId, pageId) {
-    this.widgetId = widgetId;
-    this.pageId = pageId;
-    this._self = this;
-    this.excludeLiveItems = false;
-    this.expanded = true;
-    this.hideEmptyRow = false;
-    this.defaultSortBy = 'eventDate';
-    this.defaultSearchNumber = 10;
-    this.tooltipLabels = null;
-    WcmDashboardWidgetCommon.init(this);
+CStudioAuthoringWidgets.MyRecentActivityDashboard = function (widgetId, pageId) {
+  this.widgetId = widgetId;
+  this.pageId = pageId;
+  this._self = this;
+  this.excludeLiveItems = false;
+  this.expanded = true;
+  this.hideEmptyRow = false;
+  this.defaultSortBy = 'eventDate';
+  this.defaultSearchNumber = 10;
+  this.tooltipLabels = null;
+  WcmDashboardWidgetCommon.init(this);
+
+  /**
+   * get table data
+   */
+  this.retrieveTableData = function (sortBy, sortAscDesc, callback, retrieveTableData, filterByNumber, filterBy) {
+    sortAscDesc = CStudioAuthoring.Utils.sortByAsc.init(sortBy, widgetId);
+
+    CStudioAuthoring.Service.getUserActivitiesServices(
+      CStudioAuthoringContext.site,
+      CStudioAuthoringContext.user,
+      sortBy,
+      sortAscDesc,
+      filterByNumber,
+      filterBy,
+      this.excludeLiveItems,
+      callback
+    );
+  };
 
-    /**
-     * get table data
-     */
-    this.retrieveTableData = function (sortBy, sortAscDesc, callback, retrieveTableData, filterByNumber, filterBy) {
-      sortAscDesc = CStudioAuthoring.Utils.sortByAsc.init(sortBy, widgetId);
-
-      CStudioAuthoring.Service.getUserActivitiesServices(
-        CStudioAuthoringContext.site,
-        CStudioAuthoringContext.user,
-        sortBy,
-        sortAscDesc,
-        filterByNumber,
-        filterBy,
-        this.excludeLiveItems,
-        callback
-      );
-    };
+  /**
+   * render widget specific controls
+   */
+  this.renderAuxControls = function (containerEl) {
+    var listItemEl = document.createElement('li');
+    var itemFilterEl = document.createElement('a');
 
     /**
-     * render widget specific controls
+     * adding loading image to go-live0queue widget
      */
```

Comment:
Why does this PR contain this unrelated update?

Reply:
<img width="232" alt="image" src="https://user-images.githubusercontent.com/3928341/103938169-1eac0b80-512a-11eb-8197-3fb598b6a212.png">

Please use hide whitespace chars to review